plugin/db/util/driverutil_test.go

```diff
@@ -119,3 +123,61 @@ func TestGetStatementWithResultLimit(t *testing.T) {
 		}
 	}
 }
+
+func TestApplyMultiStatements(t *testing.T) {
+	type testData struct {
+		statement string
+		total     int
+	}
+	tests := []testData{
+		{
+			statement: `
+			CREATE TABLE t(
+				a int,
+				b int,
+				c int);
+
+
+			/* This is a comment */
+			CREATE TABLE t1(
+				a int, b int c)`,
+			total: 2,
+		},
+		{
+			statement: `
+			CREATE TABLE t(
+				a int,
+				b int,
+				c int);
+
+
+			CREATE TABLE t1(
+				a int, b int c);
+			` + generateOneMBInsert(),
+			total: 3,
+		},
+	}
+
+	total := 0
+	countStatements := func(string) error {
+		total++
+		return nil
+	}
+
+	for _, test := range tests {
+		total = 0
+		err := ApplyMultiStatements(strings.NewReader(test.statement), countStatements)
+		require.NoError(t, err)
+		require.Equal(t, test.total, total)
+	}
+}
+
+func generateOneMBInsert() string {
```

Comment:
We can implement a common `GenerateRandomString(size int)` function.

Reply:
Common may not be common when it turns out. Let's make it common when it's common. (I'm not a big fan of utils and common)

Reply:
But yeah, in this case, we can remove one or the other as we merge the usage to tokenizer in my understanding.

Reply:
In this case, a common `GenerateRandomString(size int)` requires a common package in the `plugin`. I think it's unnecessary

Reply:
> I'm not a big fan of utils and common

Agreed. In this case, I am suggesting a "common" function but not necessarily in the "common" package.

Maybe a "random" package?

Reply:
IMO, this is just a helper function for unit tests. A common package for this is not necessary.